_config.yml

```diff
@@ -69,12 +70,12 @@ twitter:
   username: bitcoin_design
   card: summary
 
-
 social:
   name: Bitcoin Design
   links:
     github: bitcoindesign
     slack: bitcoindesign
+    discord: bitcoindesign
```

Comment:
it'd be good to call these "subdomains" instead of "links"

I asked Stephen what they were earlier in review but the answer wasn't clear. Glad you know!

Reply:
@GBKS No, I don't think we have a unique subdomain for our Discord instance. Not sure it's even possible.

I'm not entirely clear how that SEO jekyll tag actually maps those domains or whatever from the simple data in that JSON. I'm fine if you want to PR it out of there.

Reply:
Did a bit more Googleroni, and apparently search engines ignore this, and it should be an Array of URLs. So maybe we just delete it.